src/lib_proj.cmake

```diff
@@ -443,6 +443,16 @@ endif()
 include_directories(${SQLITE3_INCLUDE_DIR})
 target_link_libraries(${PROJ_CORE_TARGET} ${SQLITE3_LIBRARY})
 
+if(NOT DISABLE_TIFF_IS_STRONGLY_DISCOURAGED)
+  include_directories(${TIFF_INCLUDE_DIR})
+  target_link_libraries(${PROJ_CORE_TARGET} ${TIFF_LIBRARY})
+endif()
+
+if(CURL_FOUND)
+  include_directories(${CURL_INCLUDE_DIR})
```

Comment:
this should be target_include_directories

Reply:
done